### PR TITLE
Script to export users who haven't completed retrocert

### PR DESCRIPTION
**Pull request by kalvinwang**
We've used a version of this script every month for the past few months to generate a list of claimants who haven't yet completed retroactive certification. We expect it will only be used a few more times at most.

### CODE REVIEW

File: scripts/get_users_to_remind.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# The two inputs for this script are intentionally not shared
+# The users.py file is constant between runs and contains the hashes for all users
+# The forms.json file is exported directly from CosmosDB using the Data Migration Tool
```
Comment (idealisms) [inline scripts/get_users_to_remind.py:3]: Maybe include the instructions on how to generate users.py and forms.json somewhere? Could be in a markdown file in github or a google doc linked from here.

File: scripts/get_users_to_remind.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+userIdsOnCompletedForms = [i['id'] for i in data]
+
+incompleteUsers = list(set(userIds) - set(userIdsOnCompletedForms))
+
+print(len(userIds));
+print(len(userIdsOnCompletedForms));
+print(len(incompleteUsers));
+
+with open('result.txt', 'w') as file:
```
Comment (idealisms) [inline scripts/get_users_to_remind.py:19]: Maybe use the naming scheme (hashesForUsersWhoHaveNotCompletedTheForm-YYYYMMDD.txt) you've been using here so it's easy to be consistent?

**Review by idealisms — APPROVED** [review]

File: scripts/get_users_to_remind.py (hunk 1 above)
Comment (kalvinwang) [reply scripts/get_users_to_remind.py:3]: Yup it's in a private google doc that links to this script! Will share it with you / teammates

File: scripts/get_users_to_remind.py (hunk 2 above)
Comment (kalvinwang) [reply scripts/get_users_to_remind.py:19]: good idea!

**Pull request merged**